samples/v1.0/Tests/TextBlock.Test.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Can we give this file a more descriptive name?

Reply:
I will remove this card, and merge this card with one in the FactSet into a new card on a new PR.

---
In reply to: [494599048](https://github.com/microsoft/AdaptiveCards/pull/4765#discussion_r494599048) [](ancestors = 494599048)